CONTRIBUTING.md

```diff
@@ -215,21 +215,34 @@ Pull requests made against the `X-Road/develop` branch MUST follow these convent
 
 ### Commit Messages
 
-Git commit messages SHOULD follow the guidelines described below:
+Git commit messages MUST follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) guidelines.
 
-- Separate subject from body with a blank line.
-- Do not end the subject line with a period.
-- Capitalize the subject line and each paragraph.
+The commit message should be structured as follows:
+
+```
+<type>[optional scope]: <subject>
+
+[optional body]
+
+[optional footer(s)]
+```
+
+message should be formatted below:
+
+- Do not end the line with a period.
```

Comment:
Maybe `Do not end the subject line with a period.`?
It's ok to use period in body (like there's in the example below).